pkg/controller/daemon/util/daemonset_util.go

```diff
@@ -120,11 +108,7 @@ func AddOrUpdateDaemonPodTolerations(spec *v1.PodSpec, isCritical bool) {
 func CreatePodTemplate(ns string, template v1.PodTemplateSpec, generation *int64, hash string) v1.PodTemplateSpec {
```

Comment:
also remove `ns string` here :)

Reply:
I did see a reminder in one of the comments, but forgot...

Done.